### ISSUE TITLE
Chat Printing twice.

### ISSUE THREAD

**Issue by AGPSALM** [JaredScar/DiscordChatRoles#2]
![image](https://user-images.githubusercontent.com/86768183/142073054-4bd6a0fc-508c-4bcc-b686-f0414f998cf4.png)

and It dosnt work The ID's all it says is

[script:DiscordChatRo] SCRIPT ERROR: citizen:/scripting/lua/scheduler.lua:1155: No such export GetDiscordIdentifier in resource BadgerDiscordAPI

But their is no scheduler.lua in the Download

**Comment by kulathedev** [comment]
Yes, you will need to Edit or Remove the Senka Wolf Pack, I edited the lines on the server.lua (I think that was the file)

**Comment by Havoc1001** [comment]
> Yes, you will need to Edit or Remove the Senka Wolf Pack, I edited the lines on the server.lua (I think that was the file)

Do you remember how to fix this? I know its 2years ago. sorry for the reply
